CONTRIBUTING.md

```diff
@@ -1,174 +1,267 @@
-very high bar for code and API quality.
-
-- If you intend to work on a pull request, feel free to create a new branch and
-open a draft pull request early on, describing what you're working on, why you're
-working on it and what you aim to achieve with the pull request.
-	- Doing this as early as possible means we can observe and comment on your
-	work as you go, enabling us to help you or to point out potential
-	shortcomings while it's still early on and easy to rectify.
-- **Make sure your changes correspond to an approved issue.** While we appeciate
-the initiative to implement your own features in your own time, we want to make
-sure everything in Fusion is well designed and considered. Pull requests are not
-the place to suggest new features.
-	- While we heavily discourage it, we do make exceptions for some bug fixes,
-	typically trivial ones that are easy to approve. *Do not depend on this
-	process* - we reserve the right to close your pull request and manually
-	redirect you to a new or existing issue instead.
-- Avoid cosmetic changes, such as renaming, refactoring, or code formatting - if
-a pull requests attempts to change the coding convention, is centred around the
-reorganisation of code, or generally doesn't focus on stability, functionality
-or testability, it will almost always be rejected. [The Rails maintainers have a
-great explanation on why these changes are not desirable.](https://github.com/rails/rails/pull/13771#issuecomment-32746700)
-	- This includes, but is not limited to, changing the spelling of variable
-	names, changing whitespace and newlines in code, switching between tabs and
-	spaces, renaming internal functions, rearranging function parameters, and
-	regrouping code into different folders.
-- Make sure all your changes follow global and local coding convention. Make
-sure to read our [style guide document](./style-guide.md) and observe how code
-near your working area is written. Consistent code is much easier to maintain,
-and avoids style arguments.
-	- In particular, make sure that variable and function names are written to
-	the same standard as other Fusion code. Naming is where a lot of code falls
-	short, and leads to massive readability problems if left unchecked. We will
-	not hesitate to point these things out in code review.
-- When your pull request is noticed by a project maintainer, it will be subject
-to a code review.
-	- Nobody writes perfect code first time - we're all humans. You will almost
-	certainly be asked to make changes as part of this process, so please be
-	patient with us, and make sure to voice your opinion if you disagree with
-	any suggested changes or rationale so we can make your PR the best it can be!
-- Alternatively, if we decide your pull request doesn't quite align with Fusion,
+<details>
+<summary>We'll review your changes</summary>
+<ul>
+<li>Nobody writes perfect code first time - we're all humans. You will almost
```

Comment:
I'd recommend "nobody's perfect" over "we're all humans". The latter excludes those with [species dysphoria](https://en.wikipedia.org/wiki/Species_dysphoria).

Reply:
Alrighty - though I don't know anything about what you just linked (it's new to me), I don't mind rewording this for you 🙂